codegen/src/generator/rest_json.rs

```diff
@@ -115,6 +116,13 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+fn http_code_to_status_code(code: Option<i32>) -> String {
+    match code {
```

Comment:
These are printing the full response with reason: `200 OK`.  Need to keep them printing as the enum value.